### DISCUSSION TITLE
Can't complete the build process remotely because of CSS imports

### DISCUSSION THREAD

**Discussion by lnlance09** [vercel/next.js#14732] [Help]
I keep getting this error when trying to run the build process on Ubuntu. I never have a problem while testing and I'm able to complete the build process on my mac. Why is this happening? All of my CSS and SCSS imports are already in the _app.js files.

![Screen Shot 2020-06-30 at 12 08 27 PM](https://user-images.githubusercontent.com/4725880/86149930-a6a1c080-baca-11ea-8a44-4a6fa9a41779.png)

**Comment by Timer** [comment]
Could you please share a full reproduction?
